runtime/util/mz31/volatile.s

```diff
@@ -120,7 +120,7 @@ longVolatileRead_CONSTANTS    DS    0H
         lm   r0,r1,0(r3)
         lr   r3,r1
         lr   r2,r0
-        lm   r4,r15,2048(r4)                      * small frame: restor\
+        lm   r4,r15,2048(r4)                     * small frame: restore\
```

Comment:
The `e` from restore is on the next line. Similarly for the next change, and other words in this file. I think there is a line length limit at play here (72 chars), and the change will cause failure to assemble this file.

Reply:
Actually, now I see a space was removed, but the `e` on the next line should be removed.

Reply:
oh.